sdcm/nemesis.py

```diff
@@ -429,14 +429,25 @@ def _add_and_init_new_cluster_node(self, old_node_ip=None, timeout=HOUR_IN_SEC *
         self.monitoring_set.reconfigure_scylla_monitoring()
         self.set_current_running_nemesis(node=new_node)  # prevent to run nemesis on new node when running in parallel
         new_node.replacement_node_ip = old_node_ip
+        disable_autocompaction = random.choice([True, False])
+        keyspaces = self.cluster.get_test_keyspaces()
         try:
             self.cluster.wait_for_init(node_list=[new_node], timeout=timeout)
+            if disable_autocompaction:
```

Comment:
isn't this need to be before `wait_for_init` ? i.e. so it won't be disable during bootstrap ?

Reply:
this seems to be a problem, testing a solution, as we chatted earlier.

Reply:
fixed it, pushing the fixes